wizeline/wize-q-remix
# feat(embeddings): create embedding model and stored procedure

#### What does this PR do?
- ⚠️ Bumps minimum node version from 14 to 16. This is necesary for the new Prisma version, which allows using the Vector extension
- Updates docker postgres images to a fork postgres version that contains the pgvector extension. Note that Cloud SQL already supports this so it wont be necessary to update infrastructure there https://cloud.google.com/blog/products/databases/using-pgvector-llms-and-langchain-with-google-cloud-databases
- Adds find related answers Postgres stored procedure as migration
- Adds model and fields for embedding storages

#### Any background context you want to provide?
(Any information regarding the PR that the reviewer should know.)
:construction:

#### How should this be manually tested?
- Update your docker containers (might need deleting containers and re-creating).
- Run migrations, see there are no errors
- Run app and verify no errors

#### What are the relevant tickets?
- Closes #241
- Partial progress on #242, pending Terraforming to Cloud Functions and configuring to run periodically
